### DISCUSSION TITLE
Triton problem

### DISCUSSION THREAD

**Discussion by ZhuoHaoCoder** [pelicanmapping/osgearth#2526] [Support Q&A]
Recently I was trying to use osgearth's triton to achieve the ocean effect, I have tried a lot of compilation methods, but always in the magnification when stuck, someone can help me.

**Comment by ZhuoHaoCoder** [comment]
This is my earth file
![test](https://github.com/gwaldron/osgearth/assets/166690322/51753355-4049-4a04-ab1a-5ada3bbebbb9)

**Comment by lyh19930821** [comment]
1. If you have written Triton's permission in your code, delete it from your earth file.
2. If you don't have permission, use ctral+alt+. when stuck. Switch to Task Manager and tap Triton's five-minute tip. Then wait a few seconds to display.
